### ISSUE TITLE
AI's manning turrets attack own crew because of incorrect speciesname case

### ISSUE THREAD

**Issue by NilanthAnimosus** [Regalis11/Barotrauma#1102]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
Any crewmember manning guns will attack ones own crew if they are outside.

**Steps To Reproduce**
Grab a suit, swim outside, order anyone on guns - get filled with coilgun rounds or something.

**Version**
Closed alpha 0.8.9.3

**Additional information**
I have been debugging the AI in various ways trying to make them more multiplayer-viable for certain tasks.

I found this is the cause of the AI attacking their own crew with railguns:

![image](https://user-images.githubusercontent.com/29177976/52478768-53119880-2b9e-11e9-88cd-7dad93fba007.png)
In short. this needs to be lower invariant check, or for it to be "Human", as this check inside of Turret.cs 's AIOperate thinks species "Human" is not "human".

It also looks like (as a side note) the AIOperate for a turret will not factor an enemy being outside of the turret's maximum rotation limits, I think the code should be modified to account for this in target selection.
